src/_scss/pages/bulkDownload/dictionary/_dictionaryTable.scss

```diff
@@ -50,8 +82,24 @@ $dictionary-yellow: #FFF694;
                 @import "./_sorter";
             }
         }
+      
+    }
+
+    .dictionary-table__content-table {
+        margin: 0;
+    }
+
+    .dictionary-table__content {
+        overflow-x: scroll;
+        height: rem(870);
```

Comment:
Since the height here mainly limits the amount of results on the page, "max-height" should work better here